Integrations/Autodesk/maya/scripts/unityCommands.mel

```diff
@@ -25,28 +68,66 @@ global proc unityRemoveNativeMenuOnLoad(){
 }
 
 // Load a specified settings file
-proc int loadUnityFbxSettings(string $fileName, string $settingType){
+proc int loadUnityFbxSettings(string $settingType, string $optionVarName, string $defaultSettingsFileName, string $defaultSettings){
+    global string $UnityModuleName;
+
+    $fileName = `optionVar -q $optionVarName`;
+
+    // if no filename set (optionVar cleared), reset it to default filename
+    if ($fileName == 0){
+        $modulePath = `moduleInfo -moduleName $UnityModuleName -path`;
+        // {$modulePath}/scripts/{$defaultSettingsFileName}
+        $modulePath = $modulePath + "/scripts/" + $defaultSettingsFileName;
+
+        $fileName = $modulePath;
+        optionVar -stringValue $optionVarName $fileName;
+    }
+
     // check if the file exists
+    if (`file -q -ex $fileName` == false){
+        // create file with default settings
+        $fileId = `fopen $fileName "w"`;
+        fprint $fileId $defaultSettings;
+        fclose $fileId;
+    }
+
+    // if the file still doesn't exist (failed to create)
+    // print an error message.
     if (`file -q -ex $fileName` == false){
         error ("Failed to find Unity Fbx "+$settingType+" Settings at: " + $fileName);
```

Comment:
why not `eval($defaultSettings)` ?

maybe still leave a warning, but at least this way we'd have something working even in a broken installation